react-native-scripts/package.json

```diff
@@ -33,8 +33,9 @@
     "match-require": "^2.0.0",
     "minimist": "^1.2.0",
     "path-exists": "^3.0.0",
+    "progress": "^1.1.8",
```

Comment:
resolution: don't support git bash for now if you want a pleasant crna experience. powershell recommended on windows for now. if anyone reading this is interested in fixing git bash support it would be welcome!